libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/network/HttpClient.kt

```diff
@@ -18,7 +17,6 @@ import okhttp3.logging.HttpLoggingInterceptor
 import okio.buffer
 import okio.sink
 
-@Keep
```

Comment:
This was removed and we're not adding it on `libdirections-onboard` `proguard-rules.pro`, don't we need to keep it? 🤔

Reply:
`Client` isn't used anywhere, `@Keep` is not need here

Reply:
It'll be in the future, so we should add the Proguard rule or leave the `@Keep` annotation so we don't forget 😅

Reply:
Lets avoid use smt that we don't need, at least for now.
Would like to have single style on whole project and prefer `proguard` config more than `@Keep` annotation. And HttpClient will provided as a module(right?), so Proguard rule will changed. We cannot forget write rule for it, because in testing it just throw `Class not found` exception 😄

Reply:
> And HttpClient will provided as a module(right?)

Nah, don't think so.

> We cannot forget write rule for it, because in testing it just throw `Class not found` exception 😄

That's correct, maybe dropping a comment? What do you think?

Reply:
dropped a comment 👍